src/components/styles/DetailView.css

```diff
@@ -0,0 +1,39 @@
+.detail-view {
+    width: 50%; 
+    height: 100%; 
+    padding: 10px; 
+    background-color: #df095e; 
+    border-top-right-radius: 10px; 
```

Comment:
![Screenshot 2023-11-28 at 11 26 54 AM](https://github.com/bpb2008/Pokedex/assets/152167/ce5c06fb-47c8-4933-9582-45fd18db3ea2)
What about the bottom right radius?

Reply:
Whoops! Good catch!